index.html

```diff
@@ -25,39 +25,54 @@
 
   <body>
     <header>
+      <div
+        href="#"
+        class="logo show-hide"
+        role="img"
+        aria-label="Logo: Right Triangle and rhombus"
+      ></div>
       <nav>
-        <div
-          href="#"
-          class="logo"
-          role="img"
-          aria-label="Logo: Right Triangle and rhombus"
-        ></div>
-        <label class="menu-btn-container">
-          <input id="menu-toggle" type="checkbox" />
-          <div class="menu-btn"></div>
-        </label>
+        <ul class="poppins">
+          <li class="nav-item">
+            <a class="nav-link" href="#headline">Hello</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#works">Portfolio</a>
+          </li>
+          <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
+          <li class="nav-item">
+            <a class="nav-link" href="#contact">Contact</a>
+          </li>
+        </ul>
       </nav>
-      <!---------------------- headline ------------------------>
-      <section id="headline">
+      <label class="menu-btn-container">
+        <input id="menu-toggle" type="checkbox" />
+        <div class="menu-btn show-hide"></div>
+      </label>
+    </header>
+    <!---------------------- headline ------------------------>
+    <section id="headline">
+      <article class="headline-container">
         <h1 class="headline-title playfair">Alex Muñoz</h1>
         <p class="poppins">
           &nbspHey, there! I am a software developer! I can help you build a
           product, feature or website. Take a look at my work. If you like what
           you see and have a project you would like coded, don’t hesitate to
           contact me.
         </p>
-        <a href="#" class="btnBlue poppins">Start collaboration </a>
-      </section>
+        <a href="#contact" class="btnBlue poppins">Start collaboration </a>
+      </article>
       <div
-        class="shapes-design-container"
+        class="headline-deco"
         role="img"
         aria-label="Decoration: Geometric shapes mosaic"
       ></div>
-    </header>
+    </section>
+    <!---------- Projects ---------------->
     <main>
       <section id="works">
         <h1 class="playfair">Projects</h1>
```

Comment:
- [Optional} Kindly use only a single `h1` tag in your document. This is because screen readers depend on the `h1` tag to know the title of your page. Since this is your portfolio, the `h1` can be your name and every other heading should be between `h2` to `h5`. Kindly check [here](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking#:~:text=Your%20website%20should%20have%20only,tags%20in%20the%20page%20code).) for more info.

Reply:
**Hi, @Kamba56! Thanks a lot for your comment!! I want to share something with you:**
"(..this is the problem that HTML5 solves with the new sectioning elements and the new rules for the existing heading elements.)[http://diveintohtml5.it/semantics.html#:~:text=this%20is%20the%20problem%20that%20HTML5%20solves%20with%20the%20new%20sectioning%20elements%20and%20the%20new%20rules%20for%20the%20existing%20heading%20elements.]